packages/tools/examples/PolySegWasmContourToVolumeLabelmap/index.ts

```diff
@@ -146,6 +146,10 @@ async function run() {
       {
         mouseButton: MouseBindings.Primary,
       },
+      {
+        mouseButton: MouseBindings.Primary, // Shift + Left Click
+        modifierKey: KeyboardBindings.Shift,
+      },
```

Comment:
I have 90% of the changes to remove this binding. I'll get it done and create a PR after the demo.